adhocteam/Head-Start-TTADP
# Adjust AR Review page layout on smaller screens

**Description of change**

While working on #86, we noticed that the Activity Report Review page looked cramped on smaller screen sizes. This PR makes some adjustments to the layout on smaller screen widths so things are more readable.

- Stack label/values for reviewItems, and adjust font size/weight on labels at small sizes
- Adjust layout of Navigator, Navigator's SideNav for <1024 width screens

**How to test**

Open an Activity Report review page, scale the window to <1024px wide, and see how the layout changes

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/318

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
